bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -403,7 +403,7 @@ spec:
               command:
                 - bash
                 - -ec
-                - '{{ include "postgresql-ha.pgpassword" . }} psql -w -U {{ include "postgresql-ha.postgresqlUsername" . | quote }} -d {{ (include "postgresql-ha.postgresqlDatabase" .) | quote }} -h 127.0.0.1 -p {{ .Values.postgresql.containerPorts.postgresql }} -c "SELECT 1"'
+                - 'ps waux | grep "data standby clone" | grep -v grep || {{ include "postgresql-ha.pgpassword" . }} psql -w -U {{ include "postgresql-ha.postgresqlUsername" . | quote }} -d {{ (include "postgresql-ha.postgresqlDatabase" .) | quote }} -h 127.0.0.1 -p {{ .Values.postgresql.containerPorts.postgresql }} -c "SELECT 1"'
```

Comment:
What's the purpose of `grep -v grep`?

Reply:
`grep -v grep` exclude process `grep` from `ps`

Reply:
I see what you mean.. Could you try using `ps waux | grep "[d]ata standby clone"` instead?

Reply:
@juan131 I confirm, it works also.